foxy/distribution.yaml

```diff
@@ -2836,11 +2836,10 @@ repositories:
       - ros2_control
       - ros2controlcli
       - test_robot_hardware
-      - transmission_interface
       tags:
         release: release/foxy/{package}/{version}
-      url: https://github.com/bmagyar/ros2_control-release.git
-      version: 0.1.0-1
+      url: https://github.com/ros2-gbp/ros2_control-release
```

Comment:
Try adding the suffix `.git` to the end of the url. Maybe this will make CI happy.